AUTOENG-2852: lwes forwarding should use pool

Charles546 opened the pull request:
Adding pool support, since mondemand_backend_worker converts UDP to md_event, lwes backend now handles md_event instead of raw UDP structure.

djnym commented:
My only concern is that with this change we'll now be deserializing and reserializing every event we forward even if that's unnecessary.  This caused a problem when I was doing this accidentally.  However, the problem was with only a single process doing the forwarding, so this may be okay.  We'll have to test.  Also, please include a ChangeLog entry with all pull requests.

Charles546 commented:
I enhanced the mondemand_backend_worker to pass raw data instead of deserializing if it is configured with pass_raw_data = true.  I am hoping that this addresses some of your concerns.

djnym commented on ChangeLog:
```diff
@@ -1,3 +1,8 @@
+* Mon Nov 30 2015 Charles Huang <huangc.cd@gmail.com> 4.0.0
```
You should bump the version here, as well as in src/mondemand.app.src.  That way I can just merge and tag.
